uint8 robotID
uint32 red
uint32 green
uint32 blue
float32 x
float32 y
